pom.xml

```diff
@@ -151,6 +152,12 @@
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombok.version}</version>
+            <scope>provided</scope>
```

Comment:
You can remove these lines because they are in the dependencyManagement block in the parent pom.

```suggestion
```